libbeat/docs/processors-using.asciidoc

```diff
@@ -624,6 +627,90 @@ _Openstack Nova_
 }
 -------------------------------------------------------------------------------
 
+[[add-fields]]
+=== Add fields
+
+The `add_fields` processor adds additional fields to the event.  Fields can be
+scalar values, arrays, dictionaries, or any nested combination of these.  By
+default the fields that you specifiy will be grouped under the `fields`
+sub-dictionary in the event.  The `target` setting can be used to change the
+sub-dictionary used for grouping the new fields under.  To store the fields as
+top-level fields, set `target: ''`. 
+
+``target``:: (Optional) Sub-dictionary to put all fields in to. Defaults to `fields`.
+``fields``:: Fields to be added:
+
+Example:
+
+This configuration:
+
+[source,yaml]
+------------------------------------------------------------------------------
+processors:
+- add_fields:
+    target: project
+    fields:
+      name: myproject
+      id: '574734885120952459'
+------------------------------------------------------------------------------
+
+Adds these fields to any event:
+
+[source,json]
+-------------------------------------------------------------------------------
+{
+  "project": {
+    "name": "myproject",
+    "id": "574734885120952459"
+  }
+}
+-------------------------------------------------------------------------------
+
+
+[[add-labels]]
+=== Add labels
+
+The `add_labels` processors adds a set of key-value pairs to an event.
+The processor will flatten nested configuration objects like arrays or
+dictionaries into a fully qualified name by merging nested names with a `.`.
+Array entries create numeric names starting with 0.  Labels are always stored
+under the ECS compliant `labels` sub-dictionary.
```

Comment:
I wonder if users will know what we mean by "ECS compliant"?

Reply:
Hm, me too :)